src/components/Sidebar/Sidebar.story.js

```diff
@@ -33,50 +39,60 @@ const SidebarContainer = styled.div`
 `;
 
 class Container extends Component {
-  state = { selected: 0 };
+  state = { selected: 1 };
 
   changeSelected = selected => {
     this.setState({ selected });
   };
 
   render() {
     const { selected } = this.state;
-    const open = boolean('Open', true);
 
     return (
       <SidebarContainer>
         <Sidebar
-          open={open}
+          open={true}
           onClose={() => null}
           closeButtonLabel="close-button"
         >
           <Sidebar.Header>Header</Sidebar.Header>
           <Sidebar.NavList>
-            <Sidebar.Aggregator label={`Item #${1}`}>
+            <Sidebar.NavItem
+              label={`Home`}
+              selected={Number(selected) === 1}
```

Comment:
Why do you need to convert `selected` to a number? it is one already.

Reply:
This is how we're currently doing it:
https://github.com/sumup/circuit-ui/blob/b51f71be1390fe4e0e103d5f2ede4538d713f7f6/src/components/Sidebar/Sidebar.story.js#L62

...but we could remove it!